rest-api-spec/src/main/resources/rest-api-spec/api/profiling.status.json

```diff
@@ -0,0 +1,37 @@
+{
+  "profiling.status":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/observability/current/universal-profiling.html",
+      "description":"Returns basic information about the status of Universal Profiling."
+    },
+    "stability":"stable",
+    "visibility":"private",
+    "headers":{
+      "accept": [ "application/json"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_profiling/status",
+          "methods":[
+            "GET"
+          ]
+        }
+      ]
+    },
+    "params":{
+      "master_timeout":{
+        "type":"time",
+        "description":"Explicit operation timeout for connection to master node"
```

Comment:
```suggestion
        "description":"Explicit operation timeout for connections to the master node"
```

Reply:
Good point. However, for consistency reasons I like to keep it that way: this is common parameter in many API calls that is documented with this exact wording everywhere else and the user-facing documentation should be identical.

Reply:
Hm, then maybe we should fix this everywhere, but not in this PR :)